doc/interop-test-descriptions.md

```diff
@@ -60,6 +60,27 @@ Client asserts:
 *It may be possible to use UnaryCall instead of EmptyCall, but it is harder to
 ensure that the proto serialized to zero bytes.*
 
+### cacheable_unary
+
+This test verifies that gRPC requests marked as cacheable use GET verb instead
+of POST, and that server sets appropriate cache control headers for the response
+to be cached by a proxy. This interop test requires that the server is behind
```

Comment:
Stuff like "It is NOT expected to pass if client is accessing the server directly." should maybe be moved to the server-side description, just stating we don't expect most servers to support it.